tests/test_views.py

```diff
@@ -0,0 +1,38 @@
+import io
+import json
+
+
+def test_scan_no_document(client, mocker):
+    response = client.post('/scan')
+
+    assert response.status_code == 400
+
+
+def test_scan_document(client, mocker):
+    mocker.patch('app.views.clamav_scan', return_value=True)
+
+    response = client.post(
+        '/scan',
+        content_type='multipart/form-data',
+        data={
+            'document': (io.BytesIO(b'pdf file contents'), 'file.pdf')
+        }
+    )
+
+    assert response.status_code == 200
+    assert json.loads(response.get_data(as_text=True)) == {'ok': True}
+
+
+def test_scan_virus_document(client, mocker):
```

Comment:
Maybe one for test for ScanError

Reply:
I don't really know what those look like and whether we can handle any of them in a useful way. Since this will only used by Document Download at the moment I'll probably make it turn 500s into 503s and later we can look at exception logs to see what kinds of errors we get and what  we could do with them.

Reply:
Sorry I could have given you a better example. I think something like this should do the job:

`except (clamd.ClamdError, BotoClientError) as e:
            current_app.logger.exception("Scanning error file: {} {}".format(filename, e))
            return jsonify(status_code=500, "error": "Virus scan error: " + e)`

Reply:
That's more or less what the default flask error handler does though.